pkg/tracing/wire/binary.pb.go

```diff
@@ -103,18 +118,46 @@ func (m *SpanContext) GetSpanID() uint64 {
 }
 
 type Span struct {
-	Context      SpanContext `protobuf:"bytes,1,opt,name=context" json:"context"`
+	Context      SpanContext `protobuf:"bytes,1,opt,name=context,proto3" json:"context"`
 	ParentSpanID uint64      `protobuf:"varint,2,opt,name=parent_span_id,json=parentSpanId,proto3" json:"parent_span_id,omitempty"`
 	Name         string      `protobuf:"bytes,3,opt,name=name,proto3" json:"name,omitempty"`
-	Start        time.Time   `protobuf:"bytes,4,opt,name=start_time,json=startTime,stdtime" json:"start_time"`
-	Labels       []string    `protobuf:"bytes,5,rep,name=labels" json:"labels,omitempty"`
-	Fields       []Field     `protobuf:"bytes,6,rep,name=fields" json:"fields"`
+	Start        time.Time   `protobuf:"bytes,4,opt,name=start_time,json=startTime,proto3,stdtime" json:"start_time"`
```

Comment:
Why is proto3 new in some members and not others?

Reply:
Reverted, but definitely something we should investigate when we revisit the rest of the models